lib/jdomodels/src/test/java/org/sagebionetworks/repo/model/dbo/schema/JsonSchemaDaoImplTest.java

```diff
@@ -1372,6 +1372,49 @@ public void testGetDependenciesWithNullVersionId() throws JSONObjectAdapterExcep
 		assertEquals(one.getSchemaId(), dbo.getDependsOnSchemaId().toString());
 		assertNull(dbo.getDependsOnVersionId());
 	}
+	
+	@Test
+	public void testGetDependants() throws Exception {
+		int index = 0;
+		JsonSchemaVersionInfo one = createNewSchemaVersion("my.org.edu-one-1.0.0", index++);
+		// two depends on one
+		ArrayList<SchemaDependency> dependencies = new ArrayList<SchemaDependency>();
+		dependencies.add(new SchemaDependency().withDependsOnSchemaId(one.getSchemaId())
+				.withDependsOnVersionId(one.getVersionId()));
+		JsonSchemaVersionInfo two = createNewSchemaVersion("my.org.edu-two-1.0.0", index++, dependencies);
+
+		// call under test
+		List<Long> result = jsonSchemaDao.getVersionIdsOfDependants(one.getVersionId());
```

Comment:
more than one and check pagination.
Dao returns paginated list (limit offset)
Manager uses the dao/pagitation -> iterator and one message per id.